src/fidesops/schemas/privacy_request.py

```diff
@@ -22,7 +23,7 @@ class PrivacyRequestCreate(BaseSchema):
     finished_processing_at: Optional[datetime]
     requested_at: datetime
     identities: List[PrivacyRequestIdentity]
-    policy_key: str
+    policy_key: FidesKey
```

Comment:
This is one instance that isn't handled currently - an invalid policy_key in the request would cause the privacy request to fail, but this failure would be hidden because we're catching ExceptionErrors in this endpoint for some reason, so the user would have no idea why it failed.